lib/WUI/wui_REST_api.c

```diff
@@ -80,22 +80,21 @@ void get_printer(char *data, const uint32_t buf_len) {
     snprintf(data, buf_len,
         "{"
         "\"telemetry\": {"
+        "\"temp-bed\": %.1f,"
+        "\"temp-nozzle\": %.1f,"
+        "\"print-speed\": %d,"
+        "\"z-height\": %.1f,"
         "\"material\": \"%s\""
         "},"
         "\"temperature\": {"
         "\"tool0\": {"
-        "\"actual\": %d.%.1d,"
```

Comment:
Tohle jsem uz jednou opravoval, protoze to vracelo veci co nebyly cislo. Samozrejme je nejlepsi se tomu vyhnout tim %f, ale neopisuje se to pres %d protoze mame nejakou malou implementaci printf co floaty neumi?

Reply:
Samozřejmě jsem si tu změnu byl vyzkoušet. Umí je a všechno s tím funguje. Je možný, že je to o chlup _pomalejší_ než ten hack s dvěma číslama, ale přijde mi, že nemusíme řešit každou instrukci.